lib/relation-definition.js

```diff
@@ -2339,9 +2381,9 @@ EmbedsOne.prototype.destroy = function(options, cb) {
   return cb.promise;
 };
 
-RelationDefinition.embedsMany = function embedsMany(modelFrom, modelTo, params) {
+RelationDefinition.embedsMany = function embedsMany(modelFrom, modelToRef, params) {
   params = params || {};
-  modelTo = lookupModelTo(modelFrom, modelTo, params, true);
+  var modelTo = lookupModelTo(modelFrom, modelToRef, params, true);
 
   var thisClassName = modelFrom.modelName;
   var relationName = params.as || (i8n.camelize(modelTo.modelName, true) + 'List');
```

Comment:
same here: `params.reference || params.as || i8n.camelize(modelTo.pluralModelName, true);` if we decide to use reference